packages/playwright-core/src/server/deviceDescriptorsSource.json

```diff
@@ -663,6 +663,36 @@
     "hasTouch": true,
     "defaultBrowserType": "webkit"
   },
+  "iPhone 12 Mini": {
+    "userAgent": "Mozilla/5.0 (iPhone; CPU iPhone OS 14_4 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.4 Mobile/15E148 Safari/604.1",
+    "screen": {
+      "width": 375,
```

Comment:
Can you point us to the place where you got this numbers? Looking at the [official site](https://www.apple.com/iphone-12/specs/#:~:text=The%20iPhone%2012%20mini%20display,actual%20viewable%20area%20is%20less) I'd expect the screen dimensions to be 360x780.

Reply:
I double checked with an emulator, lgtm except the question in https://github.com/microsoft/playwright/pull/11429/files#r799014128.